far/FarEng.hlf.m4

```diff
@@ -2799,95 +2799,160 @@ saved path.
-    #LeftCtrl+0..9#      Go to bookmark 0..9
-
-    #Ctrl+Ins, Ctrl+C#   Copy the text highlighted as a result of
-                       the search or manual selection to the clipboard.
-    #Ctrl+U#             Remove the highlighting of the selected text.
-    #Shift+Mouse click#  Manual text selection.
-                       If the mouse cursor is over the text, the first
-                       click marks the start of the selection and the
-                       second click marks the end of the selection. Use
-                       navigation keys after the first click to bring
-                       the end position into the view. The end position
-                       may be set earlier than the start.
-
-    See also the list of ~macro keys~@KeyMacroViewerList@, available in the viewer.
+  Navigation keys
+
+  The behavior of navigation keys depends on the ~view mode~@ViewerMode@.
+
+  The following keys work in all modes:
+
+    #Up#                 ^<wrap>Line up
+    #Down#               ^<wrap>Line down
+    #PgUp#               ^<wrap>Screenful up
+    #PgDn#               ^<wrap>Screenful down
+    #Home, Ctrl+Home#    ^<wrap>Beginning of the file
+    #End, Ctrl+End#      ^<wrap>End of the file
+
+  The following additional keys work in #text mode without line wrap#:
+
+    #Left#               ^<wrap>One column left
+    #Right#              ^<wrap>One column right
+    #Ctrl+Left#          ^<wrap>20 columns left
+    #Ctrl+Right#         ^<wrap>20 columns right
+    #Ctrl+Shift+Left#    ^<wrap>Show the leftmost column
+    #Ctrl+Shift+Right#   ^<wrap>Show the rightmost column of all lines
+currently visible on the screen
+
+  In #hex# and #dump# ~view modes~@ViewerMode@, #Ctrl+Left# and
+#Ctrl+Right# keys shift the text in the window one byte at a time in the
+corresponding direction.
+
+  Viewer commands
+
+    #F1#                 ^<wrap>Help
+    #F2#                 ^<wrap>Toggle line wrap/unwrap in #text# mode, or change ~view mode~@ViewerMode@
+    #Shift+F2#           ^<wrap>Toggle wrap type (characters or words) in #text# ~view mode~@ViewerMode@
+    #F4#                 ^<wrap>Toggle ~view mode~@ViewerMode@ to #hex# and back
+    #Shift+F4#           ^<wrap>Select ~view mode~@ViewerMode@: #text#, #hex#, or #dump#
+    #F6#                 ^<wrap>Switch to ~editor~@Editor@
+    #F7#                 ^<wrap>~Search~@ViewerSearch@
+    #Shift+F7, Space#    ^<wrap>Continue search
+    #Alt+F7#             ^<wrap>Continue search in reverse direction
+    #F8#                 ^<wrap>Toggle code page (OEM/ANSI)
+    #Shift+F8#           ^<wrap>Select code page using ~menu~@CodePagesMenu
+    #Alt+F8#             ^<wrap>~Change current position~@ViewerGotoPos@
+    #Alt+F9#             ^<wrap>Toggle the size of the Far console window
+    #Alt+Shift+F9#       ^<wrap>Open ~Viewer settings~@ViewerSettings@ dialog
+    #Numpad5,F3,F10,Esc# ^<wrap>Quit viewer
```

Comment:
Interestingly, `Numpad5` with `Num Lock` ON opens search dialog instead of quitting the viewer. I'll try to fix this later.